pkgs/development/misc/avr/gcc/wrapper.nix

```diff
@@ -5,13 +5,12 @@ stdenv.mkDerivation rec {
   version = stdenv.lib.strings.getVersion avrgcc-unwrapped.name;
   nativeBuildInputs = [ makeWrapper ];
   buildInputs = [ avrgcc-unwrapped avrlibc ];
-  builder = writeText "builder.sh" ''
-    source $stdenv/setup
-    mkdir -p $out/bin $out/lib
+  buildCommand = ''
+    mkdir -p $out/bin $out
```

Comment:
The extra `$out` is not needed. (But a bit of extra pedantry never hurts :wink:)